submodules/externals/openh264

```diff
@@ -1 +1 @@
-Subproject commit 3a75956fb2584cca84a95ba1fcbc72fa2c91f98d
+Subproject commit 3067d127aa1067a04b1d7f60cef3540b25c8351e
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is 3a75956fb2584cca84a95ba1fcbc72fa2c91f98d, why would I want to use 3067d127aa1067a04b1d7f60cef3540b25c8351e?